qiskit/assembler/assemble_schedules.py

```diff
@@ -201,9 +201,11 @@ def _assemble_instructions(
             acquire_instruction_map[(time, instruction.command)].append(instruction)
             continue
 
-        if isinstance(instruction, (DelayInstruction, Delay)):
-            # delay instructions are ignored as timing is explicit within qobj
-            continue
+        if isinstance(instruction, DelayInstruction):  # deprecated
+            instruction = Delay(instruction.duration, instruction.channels[0])
+
+        if isinstance(instruction, Delay):
+            instruction = Delay(instruction.duration, instruction.channels[0])
```

Comment:
this isn't necessary:) the instruction is already in the format we want in this case :D

Reply:
Fixed in [36edd1b](https://github.com/Qiskit/qiskit-terra/pull/4233/commits/36edd1b54a563c1184bf027f3ead0a475545cd70).